test/benchmark/rendering_test.dart

```diff
@@ -0,0 +1,105 @@
+import 'dart:ui';
+
+import 'package:benchmark_harness/benchmark_harness.dart';
+import 'package:vector_tile_renderer/src/constants.dart';
+import 'package:vector_tile_renderer/src/themes/light_theme.dart';
+import 'package:vector_tile_renderer/vector_tile_renderer.dart';
+
+import '../src/test_tile.dart';
+
+Picture _renderPicture({
+  required Theme theme,
+  required double scale,
+  required Tileset tileset,
+  required double zoom,
+}) {
+  assert(scale >= 1 && scale <= 4);
+
+  double size = scale * tileSize;
+  final rect = Rect.fromLTRB(0, 0, size, size);
+
+  final recorder = PictureRecorder();
+  final canvas = Canvas(recorder, rect);
+  canvas.clipRect(rect);
+  canvas.scale(scale.toDouble(), scale.toDouble());
+
+  Renderer(theme: theme).render(
+    canvas,
+    tileset,
+    zoomScaleFactor: scale,
+    zoom: zoom,
+  );
+
+  return recorder.endRecording();
+}
+
+class RenderPicture extends BenchmarkBase {
+  RenderPicture({
+    required this.zoom,
+    this.preprocessTile = false,
+    this.preprocessThemeLayerFeatures = false,
+  }) : super('RenderPicture('
+            'zoom: $zoom, '
+            'preprocessTile: $preprocessTile, '
+            'preprocessThemeLayerFeatures: $preprocessThemeLayerFeatures'
+            ')');
+
+  static Future<void> setupAll() async {
+    testTile = await readTestTile();
+  }
+
+  static late final VectorTile testTile;
```

Comment:
is there a benefit to `static`? If possible, let's avoid it.

Reply:
I wanted to keep `RenderPicture` self-contained and the `BenchmarkBase` is sync only, so any async work needs to be done before the benchmarks are run.

Reply:
that can be addressed by adding it to `setup` below, right?